src/main/java/webapp/LoginController.java

```diff
@@ -1,14 +0,0 @@
-package webapp;
-
-
-import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.GetMapping;
-
-@Controller
-public class LoginController {
-
-    @GetMapping("/")
-    public String getLook(){
-        return "index.html";
-    }
-}
```

Comment:
IMO better option is not deleting this but made separate controller for register.